cmake/FindAvro.cmake

```diff
@@ -10,5 +10,7 @@ find_library(AVRO_LIBRARIES libavro.a)
 
 if (AVRO_INCLUDE_DIRS AND AVRO_LIBRARIES)
   message(STATUS "Found Avro C libraries: ${AVRO_LIBRARIES}")
+  set(AVRO_LIBRARIES avro)
```

Comment:
This shouldn't be needed as the AVRO_LIBRARIES variable contains the path to the library file.

Reply:
This change came from my U18.04 environment which already has libcavro in the repo.